introduction/InstruDictator.html

```diff
@@ -1,51 +1,95 @@
 {{ block title }}Instructions{{ endblock }}
 
+{{ block style }}
+
+<style>
+    /* (A) SHARED CLASS */
+    .blue_box {
+    padding: 10px;
+    border: 2px solid dodgerblue;
+    height: 40px;
+    width: 70px;
+    border-radius: 5px;
+    background: dodgerblue;
+    color: white;
+    }
+
+    div.centered {
+    width: 500px;
+    height: 400px;
+    top: 50%;
+    left: 50%;
+    position: absolute;
+    margin-top: -200px;
+    margin-left: -250px;
+}
+</style>
+
+{{ endblock }}
+
 {{ block content }}
 
 <div>
     <p>
         You are the <b>Decider</b>. This means that in every interaction you have a decision to make.
-        In each interaction, both you and the receiver are allocated with {{Constants.endowment_p1}} each.
-        You have to decide if you want to take the tokens of the receiver or leave them. If you take them,
-        you will earn {{Constants.pot_money}} in that interaction where you made that choice and the receiver gets nothing.
-        If you decide to leave it, you keep your allocated {{Constants.endowment_p1}} and so does the receiver.
+        In each interaction, both you and the receiver are allocated with <b>{{half_pot}}</b> each.
+        You have the opportunity to take the money of the receiver or leave it. If you decide to take it,
+        you will earn <b>{{pot_money}}</b> in that interaction and the receiver gets nothing.
+        If you decide to leave it, you keep your allocated <b>{{half_pot}}</b> and so does the receiver.
         The receiver will know his final amount, but not who decided.
```

Comment:
I felt this sentence was somewhat unclear. 

Perhaps this becomes more clear by saying: "The receiver will only learn the overall earnings (across all the receiver's interactions), but not the individual decisions of each decider."